bin/clover/src/pipeline-steps/createInputSocketsAcrossAssets.ts

```diff
@@ -46,11 +46,18 @@ export function createInputSocketsBasedOnOutputSockets(
           socket.data?.connectionAnnotations,
         ) as ConnectionAnnotation[];
 
-        for (const annotations of existingAnnotations) {
-          for (const annotation of annotations.tokens) {
-            foundOutputSockets[annotation] ??= [];
-            foundOutputSockets[annotation].push(schemaVariant);
+        for (const { tokens } of existingAnnotations) {
+          if (tokens.length !== 1) continue;
```

Comment:
If there is more than one token we don't want to add the schema variant for them?